src/Events/Message/index.js

```diff
@@ -0,0 +1,12 @@
+import { SOCKET_EVENTS } from '../../Constants';
+
+export const message = async ({id, text, user }, { socket, io }) => {
+  io.to(id).emit(SOCKET_EVENTS.RECEIVE_MESSAGE, {text, user});
```

Comment:
Şu anda tüm mesaj bilgileri kullanıcının tarayıcısında tutuluyor. İlerleyen zamanda bu bilgiler veritabanına da kaydedilerek sonradan odaya giren kullanıcılara yahut aynı kullanıcının tarayıcısında sayfayı yenilemesi durumunda kaybolmaması için eklenmelidir.